### PR TITLE
Add new command to support concurrent execution of ml.

### PR DESCRIPTION
**Pull request by nikunjkotecha**
Please review thoroughly, refactored a bit to reduce code duplication between the two commands drush sfml and sfmlc

Note: I did a POC in local for the code change and it works but I was not able to check the whole code on an actual environment so there is a lot of blind coding involved here.

### CODE REVIEW

File: AcsfToolsCommands.php [hunk 1]
```diff
@@ -206,72 +181,260 @@ public function ml($cmd, $command_args = '', $command_options = '', $options = [
...
+
+    foreach ($sites as $delta => $details) {
+      $domain = $this->getDomain($details, $options);
+      $process = $this->prepareCommand($domain, $details, $cmd, $command_args, $drush_command_options, $profiles);
+      if (empty($process)) {
+        continue;
+      }
+
+      $processes[$domain] = $process;
+      $this->output()->writeln("\n=> Executing command on $domain");
```
Comment (sdelbosc) [inline AcsfToolsCommands.php:282]: I guess the screen output becomes awful when we run on multiple ones at the same time

Comment (nikunjkotecha) [reply AcsfToolsCommands.php:282]: not really, output is all done in the loop below after execution is finished for particular domain
https://github.com/acquia/acsf-tools/pull/108/files#diff-16128690f3f86ae811dda9f676489477R298

Comment (sdelbosc) [reply AcsfToolsCommands.php:282]: you may have a buffer size issue then

Comment (nikunjkotecha) [reply AcsfToolsCommands.php:282]: we will have something like
executing on abc
executing on xyz

The command executed successfully for the site abc
OUTPUT
The command executed successfully for the site xyz
OUTPUT

Comment (nikunjkotecha) [reply AcsfToolsCommands.php:282]: about buffer size, any idea or link detail the issue or a way for me to reproduce?

Also, we are doing the same even today, showing full output in one go

Comment (sdelbosc) [reply AcsfToolsCommands.php:282]: I am surprised that we are doing that today but I may have missed something.
What I am worried about is getting a huge output from one command that would end up reaching a memory limit or any other limitation.
But if you are sure we do that today already we should be fine.

Comment (nikunjkotecha) [reply AcsfToolsCommands.php:282]: yep, we do that already today

**Pull request merged**